ui/src/components/RoleForm/index.jsx

```diff
@@ -255,6 +273,24 @@ export default class RoleForm extends Component {
             </SpeedDial>
           </Fragment>
         )}
+        {dialogOpen && (
+          <DialogAction
+            focusOnPrimary
+            fullScreen
```

Comment:
Given the dialog has no input fields, we don't really need to make this full screen. Let's remove `fullScreen`.